plugins/reporters/ctrlx/src/main/kotlin/CtrlXAutomationReporter.kt

```diff
@@ -26,19 +26,35 @@ import kotlinx.serialization.json.encodeToStream
 
 import org.ossreviewtoolkit.model.licenses.LicenseView
 import org.ossreviewtoolkit.plugins.api.OrtPlugin
+import org.ossreviewtoolkit.plugins.api.OrtPluginOption
 import org.ossreviewtoolkit.plugins.api.PluginDescriptor
 import org.ossreviewtoolkit.reporter.Reporter
 import org.ossreviewtoolkit.reporter.ReporterFactory
 import org.ossreviewtoolkit.reporter.ReporterInput
 import org.ossreviewtoolkit.utils.spdx.SpdxConstants
 import org.ossreviewtoolkit.utils.spdx.SpdxLicense
+import org.ossreviewtoolkit.utils.spdx.toSpdx
+
+data class CtrlXAutomationReporterConfig(
+    /**
+     * The categories of the licenses of the packages to include in the report. If a component has a license which has a
+     * category not present in this parameter, the license is removed from the component and not visible in the report.
+     * If a component has ALL its licenses removed this way, it is not displayed in the report. If the parameter is not
+     * set for the reporter, all components and all licenses are present in the report.
+     */
+    @OrtPluginOption
```

Comment:
The annotation is only required if you want to set a default value or define alias names for the option, otherwise it can be omitted (note that for nullable properties `null` is automatically used as default value, you don't have to define it).